### ISSUE TITLE
[Informative] Observable changes because of Unicode 11

### ISSUE THREAD

**Issue by mathiasbynens** [tc39/ecma262#1219]
Output of [`node compare.js`]( https://gist.github.com/mathiasbynens/c08f45efdcdb9389adf3da57e16683de):

```
$ node compare.js
Comparing Unicode v11.0.0 to Unicode v10.0.0…
Binary_Property: 6 new values
- Emoji
- Emoji_Component
- Emoji_Modifier
- Emoji_Modifier_Base
- Emoji_Presentation
- Extended_Pictographic
General_Category: 0 new values
Script: 7 new values
- Dogra (Dogr)
- Gunjala_Gondi (Gong)
- Hanifi_Rohingya (Rohg)
- Makasar (Maka)
- Medefaidrin (Medf)
- Old_Sogdian (Sogo)
- Sogdian (Sogd)
Script_Extensions: 7 new values
- Dogra (Dogr)
- Gunjala_Gondi (Gong)
- Hanifi_Rohingya (Rohg)
- Makasar (Maka)
- Medefaidrin (Medf)
- Old_Sogdian (Sogo)
- Sogdian (Sogd)
Binary_Property/ID_Start: 0 removals
Binary_Property/ID_Start: 326 new code points
Binary_Property/ID_Continue: 0 removals
Binary_Property/ID_Continue: 402 new code points
General_Category/Space_Separator: 0 removals
General_Category/Space_Separator: 0 new code points
Case_Folding=S: 0 removals
Case_Folding=S: 0 new code points
Case_Folding=F: 0 removals
Case_Folding=F: 0 new code points
```

TL;DR identifier symbols are expanded and some new properties and aliases are supported in RegExp Unicode property escapes (#1218).
